tests/e2e/pageObjects/my-redis-databases-page.ts

```diff
@@ -90,4 +100,16 @@ export class MyRedisDatabasePage {
           }
       }
   }
+
+  async clickOnEditDBByName(databaseName: string): Promise<void>{
```

Comment:
Please add a description on top (I see it didn't happen also on functions above)

Reply:
done